web/user/src/translations/es.json

```diff
@@ -0,0 +1,1054 @@
+{
+    "%s emulated": "%s emulada",
```

Comment:
This is inherited from klear, user portal was not based on it so, replace translations